### ISSUE TITLE
return http 409 if any tests fail

### ISSUE THREAD

**Issue by dmcassel** [marklogic-community/ml-unit-test#38]
This was implemented in Roxy Unit Test. See https://github.com/marklogic-community/roxy/commit/afaac5d71d1502f69d9cb44dd23401ff0c17aaa0.

**Comment by rjrudin** [comment]
I don't think a 4xx is appropriate here, as the caller didn't make any mistakes, and the server didn't encounter any errors in the processing of the request. The call itself completed successfully in terms of running the tests. The fact that a test failed is neither the caller nor the server's fault, it's a natural outcome of the test.

**Comment by dmcassel** [comment]
@rjrudin I agree with your comment. I think I filed this ticket as a difference between ml-unit-test and Roxy, where Geert had made a change to Roxy. But I agree that a failed test isn't a 4xx.
